furnace/crossref_paper.py

```
@@ -1,9 +1,21 @@
+import random
```

Ruff (0.8.2)

`random` imported but unused

Remove unused import: `random`

(F401)